### PR TITLE
vscode-extensons.github.github-vscode-theme: 1.1.5 -> 3.0.0

### PR DESCRIPTION
**Pull request by dbirks**
###### Motivation for this change
Bumping the Github theme extension to the latest version:
https://github.com/primer/github-vscode-theme/releases/tag/v3.0.0

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>vscode-extensions.github.github-vscode-theme</li>
  </ul>
</details>

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119342` at 6280bc5c run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>vscode-extensions.github.github-vscode-theme</li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`configurePhase` should probably contain `runHook preConfigure` and `runHook postConfigure`.

Near pkgs/misc/vscode-extensions/vscode-utils.nix:21:12:
```
   |
21 |     inherit configurePhase buildPhase dontPatchELF dontStrip;
   |            ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/misc/vscode-extensions/vscode-utils.nix:21:12:
```
   |
21 |     inherit configurePhase buildPhase dontPatchELF dontStrip;
   |            ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

unzip is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/misc/vscode-extensions/vscode-utils.nix:25:5:
```
   |
25 |     buildInputs = [ unzip ] ++ buildInputs;
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/misc/vscode-extensions/default.nix [hunk 1]
```diff
@@ -412,9 +412,9 @@ let
           mktplcRef = {
             name = "github-vscode-theme";
             publisher = "github";
-            version = "1.1.5";
+            version = "3.0.0";
             sha256 =
-              "10f0098cce026d1f0c855fb7a66ea60b5d8acd2b76126ea94fe7361e49cd9ed2";
+              "1a77mbx75xfsfdlhgzghj9i7ik080bppc3jm8c00xp6781987fpa";
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:417]:
```suggestion
```

File: pkgs/misc/vscode-extensions/default.nix [hunk 2]
```diff
@@ -412,9 +412,9 @@ let
           mktplcRef = {
             name = "github-vscode-theme";
             publisher = "github";
-            version = "1.1.5";
+            version = "3.0.0";
             sha256 =
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:416]:
```suggestion
            sha256 = "1a77mbx75xfsfdlhgzghj9i7ik080bppc3jm8c00xp6781987fpa";

```

File: pkgs/misc/vscode-extensions/default.nix [hunk 3]
```diff
@@ -412,9 +412,9 @@ let
           mktplcRef = {
             name = "github-vscode-theme";
             publisher = "github";
-            version = "1.1.5";
-            sha256 =
-              "10f0098cce026d1f0c855fb7a66ea60b5d8acd2b76126ea94fe7361e49cd9ed2";
+            version = "3.0.0";
+            sha256 = "1a77mbx75xfsfdlhgzghj9i7ik080bppc3jm8c00xp6781987fpa";
+
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:417]:
```suggestion
```

**Comment by dbirks** [comment]
Thanks for the fixes Sandro, and for the merge :+1:

**Pull request merged**